src/Maintenance/Maintenance/Az.Maintenance.psd1

```diff
@@ -115,10 +115,10 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Got ''Az-MaintenanceConfiguration'' will honor filters while listing maintenance configurations'
```

Comment:
There is no need to change the ReleaseNotes manually. You should change the `changelog.md` and our tools will change ReleaseNotes.